docs/static/templates/visualization-template.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Visualization title
+menuTitle: Visualization
+description: Use this template when you write a visualization topic.
+aliases:
+  - /docs/writers-toolkit/latest/templates/concept-template
+weight: 100
+keywords:
+  - keyword 1
+  - keyword 2
+  - keyword 3
+---
+
+<!-- Refer to [Front matter]({{< relref "../../front-matter/" >}}) for more information about how to populate front matter. -->
+
+# Visualization title
+
+<!-- The visualization title is required. This is the name of the visualization as it appears in the UI. For example: Time series.
+
+A visualization topic provides an overview to help end users understand how to best use the visualization type and the options available to them in the user interface (UI). Visualization topics always include conceptual and reference content and may include task content.
+
+Refer to the [Visualization topic documentation](https://grafana.com/docs/writers-toolkit/writing-guide/topic-types/visualization/) for guidelines on writing a visualization topic.
+-->
+
+Introduce the concept.
+
+<!-- The introduction is required. Add an introduction to the visualization that explains what visualization type can help the user do. -->
+
```

Comment:
```suggestion

Main visual representing the visualization. 

<!-- A screenshot is generally preferred because it's easiest to maintain, but a five-to-ten second video is acceptable. Please don't use longer videos as the main visual aid here. They can be added elsewhere in the page, if needed. -->

```